<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddInfluenceLoadsDialog</class>
 <widget class="QDialog" name="AddInfluenceLoadsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Influence Loads</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="selectedJointsLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Selected Joints</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="pointLoadsLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Point Loads</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QTextEdit" name="selectedJointsTextEdit">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="3">
    <widget class="Line" name="lineA">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="3">
    <widget class="QListWidget" name="pointLoadsListWidget"/>
   </item>
   <item row="1" column="3" rowspan="3">
    <widget class="Line" name="lineB">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QGroupBox" name="directionGroupBox">
     <property name="title">
      <string>Direction</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="leftToRightRadioButton">
        <property name="text">
         <string>LR</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rightToLeftRadioButton">
        <property name="text">
         <string>RL</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="4" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="pointLoadLabel">
       <property name="text">
        <string>V</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="pointLoadDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="offsetLabel">
       <property name="text">
        <string>Offset</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="offsetDoubleSpinBox"/>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacerA">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="addPointLoadPushButton">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../../rsc.qrc">
         <normaloff>:/icons/list-add.png</normaloff>:/icons/list-add.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="horizontalSpacerB">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="removePointLoadPushButton">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="../../rsc.qrc">
         <normaloff>:/icons/list-remove.png</normaloff>:/icons/list-remove.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="horizontalSpacerC">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="addInfluenceLoadPushButton">
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset resource="../../rsc.qrc">
         <normaloff>:/icons/dialog-ok-apply.png</normaloff>:/icons/dialog-ok-apply.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../rsc.qrc"/>
 </resources>
 <connections/>
</ui>
